### PR TITLE
Use Ubuntu 18.04 in Vagrant

### PR DESCRIPTION
**Pull request by awangc**
- Update vagrant box, omit kernel upgrade, update packages needed.

### CODE REVIEW

File: vm-setup.sh [hunk 1]
```diff
@@ -5,8 +5,8 @@ export WIRESHARK_VERS=2.6.4
 
 # Install dependencies
 sudo apt-get -q update
-sudo apt-get -q install -y cmake mg make git linux-headers-$(uname -r) clang-format \
-                           pkg-config linux-image-extra-$(uname -r) xauth xterm \
+sudo -E apt-get -q install -y cmake mg make git linux-headers-$(uname -r) clang-format \
+	                   pkg-config linux-modules-extra-$(uname -r) xauth xterm \
```
Comment (zeeshanlakhani) [inline vm-setup.sh]: minor: pkg-config under cmake and aligned that way.

**Comment by zeeshanlakhani** [comment]
@awangc I'll hold off final review, as we'll want to make sure travis is updated. I think the only change is to https://github.com/williamofockham/utils/blob/master/.travis.yml#L2, which should be simple (if everything builds!)

**Comment by awangc** [comment]
@zeeshanlakhani  :+1: netbricks tests pass. 
```
RUNNING: signals
nohup: appending output to 'nohup.out'
----
[WARN] SIGHUP.
[WARN] SIGTERM.
PASS
```

**Comment by zeeshanlakhani** [comment]
:shipit:

**Pull request merged**